ui/helpers/utils/settings-search.test.js

```diff
@@ -138,6 +138,10 @@ const t = (key) => {
       return 'Show balance and token price checker';
     case 'currencyRateCheckToggleDescription':
       return 'We use Coingecko and CryptoCompare APIs to display your balance and token price. Privacy Policy';
+    case 'watch-only':
+      return 'Watch Ethereum Accounts (Beta)';
+    case 'bitcoin-support':
+      return 'Enable "Add a new Bitcoin account (Beta)"';
```

Comment:
Kind of unrelated to that PR, but the wording did change with the new design with multi-SRP support:

![Screenshot 2025-03-25 at 12 06 42](https://github.com/user-attachments/assets/d85d8880-4dd8-4d2e-bddc-21615b3c0a83)

So the text is not "Add a new Bitcoin account..." anymore (we might want to change that in the settings tab too)

I do think we can change that in another PR though.